### PR TITLE
Css/image diversity

### PR DESCRIPTION
**Pull request by ColtBlood**

### CODE REVIEW

File: public/images/reusables/buttercup/avatar.webp
Comment (JesseBrand) [inline public/images/reusables/buttercup/avatar.webp:1]: Buttercup is a PC, can you move him to the appropriate folder?

File: public/images/reusables/elves/captain-of-the-guard.jpeg
Comment (JesseBrand) [inline public/images/reusables/elves/captain-of-the-guard.jpeg:1]: Is this related to a specific session? Then move to sessions folder. If it is to represent the character (Myacine), I have her portrait instead

File: public/images/reusables/pcs/aboouck/kitchen.jpeg
Comment (JesseBrand) [inline public/images/reusables/pcs/aboouck/kitchen.jpeg:1]: Should this not be in an appropriate session folder?

File: public/images/reusables/buttercup/avatar.webp
Comment (ColtBlood) [reply public/images/reusables/buttercup/avatar.webp:1]: done

File: public/images/reusables/elves/captain-of-the-guard.jpeg
Comment (ColtBlood) [reply public/images/reusables/elves/captain-of-the-guard.jpeg:1]: done

File: public/images/reusables/pcs/aboouck/kitchen.jpeg
Comment (ColtBlood) [reply public/images/reusables/pcs/aboouck/kitchen.jpeg:1]: done

**Review by JesseBrand — APPROVED** [review]
